Rakefile

```
@@ -5,6 +5,8 @@ require "erb"
 require "fileutils"
 require "timeout"
 
+SUPPORTED_RUBY_VERSIONS = ["2.3.8", "2.4.5", "2.5.3", "2.6.1"]
```

This list is specific to Kokoro, not a statement on what is supported by the gems. Can we rename so the intended purpose is more clear?

```suggestion
KOKORO_RUBY_VERSIONS = ["2.3.8", "2.4.5", "2.5.3", "2.6.1"]
```

---

SGTM